src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Modules.java

```diff
@@ -1648,7 +1648,7 @@ private void initAddExports() {
             if (!isValidName(packageName))
                 continue;
 
-            if (!allowAccessIntoSystem && (msym.flags() & Flags.SYSTEM_MODULE) != 0) {
+            if (!allowAccessIntoSystem && msym.isFlagSet(TypeSymbolFlags.SYSTEM_MODULE)) {
                 log.error(Errors.AddExportsWithRelease(msym));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.